tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Catalogue Solutions Section Update.feature

```diff
@@ -0,0 +1,49 @@
+﻿Feature: Update the catalogue solutions section for an order
+    As a Buyer User
+    I want to update the catalogue solutions section for a given order
+    So that I can complete my order
+
+Background:
+    Given Orders exist
+        | OrderId    | Description         | OrderStatusId | Created    | LastUpdated | LastUpdatedBy                        | OrganisationId                       |
+        | C000014-01 | Some Description    | 1             | 11/05/2020 | 11/05/2020  | 335392e4-4bb1-413b-9de5-36a85c9c0422 | 4af62b99-638c-4247-875e-965239cd0c48 |
+    And the user is logged in with the Buyer role for organisation 4af62b99-638c-4247-875e-965239cd0c48
+
+@4631
+Scenario: 1. Update an orders catalogue solutions
+    When the user makes a request to update the order catalogue solutions section with the ID C000014-01
+    Then a response with status code 204 is returned
+    And the order with ID C000014-01 has catalogue solutions viewed set to true
+    And the order with orderId C000014-01 is updated in the database with data
+        | LastUpdatedBy                        | LastUpdatedByName |
+        | 7b195137-6a59-4854-b118-62b39a3101ef | Bob Smith         |
+    And the order with orderId C000014-01 has LastUpdated time present and it is the current time
+
+@4631
```

Comment:
Should this be @7840?

Reply:
I think story 4631 fits with these tests. The change is in this PR because I renamed the file. I will leave these tags as they are.